crates/extensions/c8y_config_manager/src/config.rs

```diff
@@ -95,6 +95,23 @@ impl ConfigManagerConfig {
             mqtt_port,
             tedge_http_address,
             tedge_http_port,
-        ))
+        );
+        is_config_dir_exist(config.plugin_config_path.as_path())?;
+        Ok(config)
+    }
+}
+
+fn is_config_dir_exist(config_dir: &Path) -> Result<(), TEdgeConfigError> {
```

Comment:
```suggestion
fn plugin_config_dir_exists(config_dir: &Path) -> Result<(), TEdgeConfigError> {
```
Just to avoid any confusion with `tedge_config_dir`

Reply:
Used existing function from `tedge-utils`